packages/nimble-components/src/icon-base/tests/icons.mdx

```diff
@@ -21,4 +21,4 @@ import * as iconsStories from './icons.stories';
 
 ## Accessibility
 
-When using an icon standalone, the `title` attribute should usually be set, to provide accessible text (which will also show as a tooltip).
+When using an icon standalone, and it has a meaning that is not otherwise communicated by adjacent text, set the `role` attribute to `img` and the `aria-label` attribute to a descriptive label that should be used as the accessible name. When slotting an icon into another component, if that component's accessible name fully describes its semantics, it is not neccessary to give the icon its own role and accessible name. For example, when slotting a "+" icon into a button labeled "Add File", the button's label serves as the accessible name and fully describes the button's semantics. In that case, the "+" icon is purely decorative. However, if the button's label was just "File", it would be neccessary to set the icon's role and its `aria-label` to "Add".
```

Comment:
How come we're getting rid of the guidance to set `title`? If the icon is being used completely standalone (i.e. not slotted into a button or other input), wouldn't we still want a way for mouse users to learn its meaning?

Perhaps this is a moot point since we'd almost never recommend placing an icon in the middle of nowhere with no other context. The only use case I can think of is an icon table column, but that already has guidance to set user-visible text for tooltips and group labels via its custom API.

How about something like this?

```suggestion
An icon's image should never be the sole way for a user to understand its meaning. Always provide additional context via adjacent visible text or via a `title` for sighted mouse users and a `role` and `aria-label` for users of accessibility technology. Set the `role` attribute to `img` and the `aria-label` and `title` attributes to a descriptive label.

For example, when slotting a "+" icon into a button labeled "Add File", the button's label serves as the accessible name and fully describes the button's semantics. In that case, the "+" icon is purely decorative. However, if the button's label was just "File", it would be neccessary to set the icon's role and its `aria-label` to "Add".
```

Reply:
I had removed the guidance about `title` because if we are telling them to use `aria-label`, it is no longer an accessibility concern to set `title`. It may still be a good idea to set title (for sighted users), but I didn't think we needed to call that out in our accessiblity section.

I'm fine with your version, though.